.github/workflows/ci.yml

```
@@ -0,0 +1,230 @@
+            echo "env_changed=true" >> $GITHUB_OUTPUT
+          else
+            echo "env_changed=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: 💬 Comment on PR if env changed
+        if: steps.env_check.outputs.env_changed == 'true'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const comment = `⚠️ **Environment Configuration Changed**
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)